types/index.d.ts

```diff
@@ -0,0 +1,61 @@
+declare module 'machina' {
+  interface BehavioralFsmInstance<S> {
+    __machina__?: {
+      [namespace: string]: {
+        targetReplayState: S;
+        state: S;
+        priorState: S;
```

Comment:
`targetReplayState`, `state`, and `priorState` are strings. `state`, `priorState` and `currentActionArgs` are optional.

As below, I think these mistakes are also probably traceable back to me.